spec/spec_helper.rb

```diff
@@ -61,8 +61,9 @@
 
   config.fail_fast = false
 
-  if ENV['VCR_ON'].nil? || ENV['VCR_ON'].downcase == 'true' # VCR is enabled by default - set VCR_ON=false to disable
-    require 'webmock/rspec'
+  # if ENV['VCR_ON'].nil? || ENV['VCR_ON'].downcase == 'true' # VCR is enabled by default - set VCR_ON=false to disable
+  # remove this when VCR is fixed
+  if false
```

Comment:
warning: Lint/LiteralInCondition: Literal `false` appeared in a condition. (rubocop)